LiskHQ/lisk-sdk
# Incorrect schema and types used in interoperability

### What was the problem?

This PR resolves #8318 #8319

### How was it solved?

- Added correct schema for endpoint request params- [🐛](https://github.com/LiskHQ/lisk-sdk/commit/01bc6792e3eac09207d6507fbaff54682b8577e8) [Incorrect schema for endpoint -](https://github.com/LiskHQ/lisk-sdk/commit/01bc6792e3eac09207d6507fbaff54682b8577e8) [Closes](https://github.com/LiskHQ/lisk-sdk/commit/01bc6792e3eac09207d6507fbaff54682b8577e8) https://github.com/LiskHQ/lisk-sdk/issues/8318
- Added correct type and removed access to invalid field - [🐛](https://github.com/LiskHQ/lisk-sdk/commit/035a3129683a6847d2bb24d8f80a0aaa3f80a098) [Invalid access to generatorKey property -](https://github.com/LiskHQ/lisk-sdk/commit/035a3129683a6847d2bb24d8f80a0aaa3f80a098) [Closes](https://github.com/LiskHQ/lisk-sdk/commit/035a3129683a6847d2bb24d8f80a0aaa3f80a098) https://github.com/LiskHQ/lisk-sdk/issues/8319
- Bumped versions on example apps


### How was it tested?

- Follow the [guide](https://github.com/LiskHQ/lisk-sdk/blob/development/examples/interop/README.md) to setup sidechain and mainchain, get your chain registered and observe CCU transactions happening
- Call endpoint `interoperability_getChainAccount` to check registered chains on each app
- Call endpoint `chainConnector_getSentCCUs` to check if CCUs are being sent successfully from chain connector
